earth_enterprise/SConstruct

```diff
@@ -134,21 +119,54 @@ env = khEnvironment(ENV={'PATH' : '/bin:/bin:/usr/bin'},
                     exportdirs=exportdirs,
                     installdirs=installdirs,
                     JAVA_HOME="/opt/google/java")
-env.Alias('install', installdirs['root'])
+
+def getArgumentVarsAsString():
+  vars=''
+  for var, value in ARGUMENTS.iteritems():
+    if var != 'installdir':
+      vars += var + '=' + str(value) + ' '
+  return vars[:-1]  # Remove the trailing space
+
+arg_vars = getArgumentVarsAsString()
+num_jobs = GetOption('num_jobs')
+
+# Build Fusion and Server
+build_cmds = {'build': [
+    'cd src; scons -j%d %s third_party' % (num_jobs, arg_vars),
+    'cd src; scons -j%d %s' % (num_jobs, arg_vars)
+]}
+run_build = env.PhonyTargets(**build_cmds)
+
+# Copy the binaries to the install staging directory
+stage_bin_cmds = {'stage_bin': [
+    'cd src; scons -j%d installdir=%s %s install' % (num_jobs, installdir, arg_vars)
```

Comment:
Is this going to work, if `installdir` contains spaces, or other special characters?